examples/wesleycsj-hello-world.cpp

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
Nice! Now you should delete the old file. The PR still has the older file.

Reply:
Done, plus. Fixed case in commit message.